### ISSUE TITLE
[Question] Javascript based UI improvements (my own pull request)

### ISSUE THREAD

**Issue by ghost** [iv-org/invidious#3209]
Just wanted to ask if such additions would be accepted from my own pull request first.
One that I could think of at the moment would be for playlist items to able to reordered through dragging.
I would also like to know if I should not use JS at all if I wanted to add UI improvements (use CSS instead)?

**Comment by unixfox** [comment]
There are plenty of existing features in invidious that only work with JS enabled, I mean I don't see the issue to add more.

If for whatever reason you can't do it without JS or you don't want to do it without JS that's okay as long as you are not modifying existing features in order to make them working only with JS.

tldr send you PR and we will discuss about it in the PR.
